### PR TITLE
chore: upgrade bootstrap to 3.4.1

### PR DESCRIPTION
**Pull request by ert78gb**

### CODE REVIEW

**Comment by mondalaci** [comment]
The width of the tooltips have changed. The following is not good:



Other tooltips have changed, too. Please fix them according to the following screenshots:



(On the above image, the tooltip is transparent. Please make it opaque like the other tooltips.)




Please fix the following, too:

**Comment by ert78gb** [comment]
I fixed the width problem of the tooltips. I can not reproduce the opaque problem.

File: packages/uhk-web/src/app/directives/tooltip/tooltip.directive.ts [hunk 1]
```diff
@@ -56,6 +60,12 @@ export class TooltipDirective implements AfterContentInit, OnChanges {
             innerStyle = `style="max-width: ${this.maxWidth}px;"`;
         }
 
+        console.log({
```
Comment (mondalaci) [inline packages/uhk-web/src/app/directives/tooltip/tooltip.directive.ts]: Why do you want to log this?

Comment (ert78gb) [reply packages/uhk-web/src/app/directives/tooltip/tooltip.directive.ts]: I just used it for ugly debugging. Now deleted

**Pull request merged**